### ISSUE TITLE
isSymbol

### ISSUE THREAD

**Issue by char0n** [char0n/ramda-adjunct#806]
Checks if value is classified as a Symbol primitive or object.

**Possible implementation**:
```js
const isSymbol =  curry1((val) => {
    return typeof val === 'symbol'
        || typeof val === 'object' && type(val) === 'Symbol';
});
```
[type](https://ramdajs.com/docs/#type) function is from ramda.

[This PR](https://github.com/char0n/ramda-adjunct/pull/1067) should serve as a model template of PR for `isSymbol` function.

**Comment by peaonunes** [comment]
Hi @char0n . I was hoping to tackle this issue, but I am getting an error while trying to `npm install`. Seems like the version of `sinon` in the `package-lock.json` is not available anymore 🤔 . Any thoughts on that?
```bash
npm ERR! code E404
npm ERR! 404 Not Found: sinon@7.4.0
```

**Comment by char0n** [comment]
@peaonunes let me check, and I'l get back to you in a couple of minutes.

**Comment by byurhanbeyzat** [comment]
Hi, @char0n I have the same problem.

**Comment by char0n** [comment]
Fixed by db827c5d43f9a0443d21607f0330f3cea5aade17

**Comment by peaonunes** [comment]
No worries!
I'm going to tackle this issue soon 😄
